pricing-template-dotnet/README.md

```diff
@@ -58,7 +58,7 @@ Follow the instructions below to create a new adapter project based on the Prici
         }*/
         ```
 
-    1.  In this project's `build.gradle` add artifactory as a dependency source :
+    2. In this project's `build.gradle` add artifactory as a dependency source :
```

Comment:
Actually, I think this needs to be updated too

Reply:
I talked to Weish and we will move the dotnet template work to a different ticket